apollo-compiler/src/main/kotlin/com/apollographql/apollo3/compiler/codegen/java/model/ModelBuilder.kt

```diff
@@ -103,4 +113,43 @@ internal class ModelBuilder(
       }
     }
   }
+
+  private fun TypeSpec.addBuilder(context: JavaContext): TypeSpec {
```

Comment:
Ported from [testUtils#withBuilder](https://github.com/apollographql/apollo-kotlin/blob/66cd44629e374b667d16ee831f5a4d42cc38c50b/apollo-compiler/src/main/kotlin/com/apollographql/apollo/compiler/testUtils.kt#L364). Moved here since now it's only needed in one place, in this class.